.github/workflows/main.yml

```diff
@@ -59,7 +59,7 @@ jobs:
         fetch-depth: 50
     - name: Start PostgreSQL
       working-directory: docker
-      run: docker-compose up -d && docker-compose logs
+      run: docker-compose down && PGV=10 XA=yes SCRAM=yes docker-compose up -d
```

Comment:
NIT: Is there a way to put comments in a yaml? Is it worth explaining what and why the command is like this

docker-compose logs not needed anymore?

Reply:
I don't think we are able to access the logs, so it isn't very helpful to log it.